src/components/cardbuilder/card.css

```diff
@@ -271,6 +271,10 @@ button {
     position: relative;
 }
 
+.visualCardBox-cardFooter {
+    height: 5.5em;
```

Comment:
Instead of just setting a larger value for the height of the footer and hoping the content will fit within the larger height, perhaps we should just change the `height: 3.5em;` of `.cardFooter` to a `min-height`. That will allow the footer to expand dynamically based on the size of the content.

Reply:
![Screenshot_2019-09-27 Jellyfin(2)](https://user-images.githubusercontent.com/3450688/65779006-ec34f580-e114-11e9-9726-50382a510ac5.png)

Reply:
Not a fan of the different sizes. I think the original method works fine until we can figure out a way to make these look half decent. They're absolutely terrible right now.

Reply:
I agree it would be nice if they were a consistent size, but this change doesn't _fix_ the problem here. It only makes it less likely to happen, if the footer content is too large (taller than `5.5em`) it will still overflow the container. Even if we increase the size, we should still use `max-height` to avoid this or hide the overflow. To properly fix this so all cards are equal in size even if the content of one is larger would probably need a more extensive overhaul.

Reply:
@thornbill done